src/components/user-defined-sets/NamedSetManager.test.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Adapter from 'enzyme-adapter-react-16';
+import { shallow, configure } from 'enzyme';
+import expect from 'expect';
+import NamedSetManager from './NamedSetManager';
+import * as Sets from './sets';
+
+configure({ adapter: new Adapter() });
+
+describe('NamedSetManager.js', () => {
+  describe('<NamedSetManager />', () => {
+    it('renders name and edit button when not editing', () => {
```

Comment:
The test coverage you has as-is is great, but I don't think it actually does what you say it does? The `expect` just checks that there is no `<input>`. I trust your judgement about what is enough enough testing, but make sure the summary doesn't over-promise.